app/controllers/follows_controller.rb

```diff
@@ -53,16 +53,14 @@ def bulk_show
   def create
     authorize Follow
 
-    followable = case params[:followable_type]
-                 when "Organization"
-                   Organization.find(params[:followable_id])
-                 when "Tag"
-                   Tag.find(params[:followable_id])
-                 when "Podcast"
-                   Podcast.find(params[:followable_id])
-                 else
-                   User.find(params[:followable_id])
-                 end
+    followable_klass = case params[:followable_type].capitalize
+                       when "Organization", "Tag", "Podcast"
+                         params[:followable_type].constantize
```

Comment:
I made a mistake, left this one out.

```suggestion
                         params[:followable_type].capitalize.constantize
```